src/script/entity/message/File.ts

```diff
@@ -31,7 +31,10 @@ import {AssetTransferState} from '../../assets/AssetTransferState';
 import {AssetType} from '../../assets/AssetType';
 import {AssetUploadFailedReason} from '../../assets/AssetUploadFailedReason';
 
-type AssetMetaData = ProtobufAsset.IAudioMetaData | ProtobufAsset.IImageMetaData | ProtobufAsset.IVideoMetaData;
+type AssetMetaData = (ProtobufAsset.IAudioMetaData | ProtobufAsset.IImageMetaData | ProtobufAsset.IVideoMetaData) & {
+  loudness?: number[];
+  duration?: number;
+};
```

Comment:
Why do you need this? We are relying on Protobuf types here, and `loudness` or `duration` don't exist as keys in the protobuf spec.

Reply:
These are added later by us in the cryptographyMapper. This isn't ideal but we have to work with that for now.